src/components/ckeditor/ckeditor.scss

```diff
@@ -51,3 +51,12 @@ div.copy-divider.dark {
 div.copy-divider.maroon {
   background-color: #8c1d40;
 }
+
+drupal-media {
```

Comment:
There's no `.` or `#` in front of this selector. Is that right, @ovista-dept ?

Reply:
Ohhhhh, it's a tag! Wasn't familiar with that drupal-media tag usage. Cool. Thanks!